web/src/beta/features/Dashboard/ContentsContainer/Members/index.tsx

```diff
@@ -76,37 +115,60 @@ const Members: FC<{ currentWorkspace?: Workspace }> = ({
 
 export default Members;
 
-const Wrapper = styled("div")(() => ({
+// styled components
+const Wrapper = styled("div")({
   display: "grid",
-  gridTemplateRows: "auto 1fr",
-  boxSizing: "border-box",
+  gridTemplateRows: "auto auto 1fr",
   height: "100%"
-}));
+});
+
+const HeaderWrapper = styled("div")({
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center"
+});
 
 const Search = styled("div")(({ theme }) => ({
   display: "grid",
-  gridTemplateColumns: "1fr 1fr",
+  gridTemplateColumns: "2fr 1fr",
   gap: theme.spacing.super,
-  width: "max-content",
-  padding: theme.spacing.largest,
-  "@media (min-width: 650px)": {
-    maxWidth: "550px",
-    gridTemplateColumns: "2fr 1fr"
-  }
+  padding: theme.spacing.largest
 }));
 
 const ListWrapper = styled("div")(({ theme }) => ({
   display: "flex",
   flexDirection: "column",
   gap: theme.spacing.normal,
   overflowY: "auto",
-  boxSizing: "border-box",
-  padding: `0 ${theme.spacing.largest}px ${theme.spacing.largest}px ${theme.spacing.largest}px`
+  padding: theme.spacing.largest
 }));
 
-const TemplateWrapper = styled("div")(() => ({
+const TemplateWrapper = styled("div")({
   display: "flex",
   alignItems: "center",
   justifyContent: "center",
   height: "70vh"
+});
+
+const Table = styled("div")(({ theme }) => ({
+  padding: `${theme.spacing.smallest}px 32px`,
+  display: "grid",
+  gridTemplateColumns: "0.1fr 2.9fr 4fr 2fr 1fr",
+  gap: theme.spacing.small,
+  color: theme.content.main,
+  height: "28px"
+}));
+
+const TableHeaderCell = styled("div")(({ theme }) => ({
+  "&:nth-child(2)": {
+    transform: "translateX(-28px)"
+  },
+  "&:nth-child(3)": {
+    transform: "translateX(2px)"
+  },
```

Comment:
Try avoid using `:nth-child`, use `:nth-of-type` instead. BTW why is this needed since you already set table as grid and has very customized `gridTemplateColumns: "0.1fr 2.9fr 4fr 2fr 1fr",` in use?